src/openrct2-ui/windows/News.cpp

```diff
@@ -92,13 +93,14 @@ class NewsWindow final : public Window
 
         size_t j = _pressedNewsItemIndex;
         _pressedNewsItemIndex = -1;
+        auto& gameState = OpenRCT2::GetGameState();
```

Comment:
Could you put `using namespace OpenRCT2;` just under the `#include`s and replace `OpenRCT2::GetGameState()` with just `GetGameState()`?